apps/web/src/components/nav/NavMenuButton/NavMenuLinkButton.tsx

```diff
@@ -1,7 +1,7 @@
 import { FC, PropsWithChildren, useState } from 'react';
 import { NavLink } from 'react-router-dom';
-import { css } from '../../../styled-system/css';
-import { HStack } from '../../../styled-system/jsx';
+import { css } from '@novu/novui/css';
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** ‏Example imports using the new DS